index.js

```diff
@@ -32,22 +30,31 @@ class KeyState extends q.DesktopApp {
   // eslint-disable-next-line class-methods-use-this
   async run() {
     KeyState.log('Running...');
+
+    this.pollingInterval = this.config.pollingInterval || 1000;
+    this.colors = {
+      on: this.config.colorOn || '#FF0000',
+      off: this.config.colorOff || '#00FF00',
+    };
```

Comment:
Was this moved to allow config updates while it's running?  If so, that's great.

Reply:
Oh sorry I misread your description around this change.  Thanks for fixing that.